ckanext/canada/templates/internal/user/login.html

```diff
@@ -52,7 +52,7 @@
     {% endif %}
 
     {% block form %}
-      {% snippet 'user/login_form.html', action=c.login_handler, error_summary=error_summary %}
+      {% snippet 'user/snippets/login_form.html', action=c.login_handler, error_summary=error_summary %}
```

Comment:
In /usr/lib/ckan/default/src/ckan/ckan/templates/user/login.form, it uses such variable. Do you mean I should follow the one in wet-boew, as `error_summary=errors` ?